tests/pom.xml

```diff
@@ -320,6 +320,40 @@
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>jacoco</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
+            <version>${jacoco.version}</version>
+            <!-- integration tests shouldn't use PowerMock, so recommended prepare-agent goal can be used, instead of offline-instrumentation -->
```

Comment:
Have you verified that none of them currently use PowerMock?

Reply:
Yes, there are no PowerMock usage in integration tests. There are usages of whitebox however, but I don't think those cause problems with the code coverage.